### PR TITLE
Feature/#24 Created EmailSenderService

### PR DESCRIPTION
**Pull request by botirjonabdumannonovv**
- Created IEmailSenderService interface and Implemented in EmailSenderService

### CODE REVIEW

**Review by Habiba67y — DISMISSED** [review]

**Review by Mirzohid-Mominov — DISMISSED** [review]

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Emails/Interfaces/IEmailSenderService.cs [hunk 1]
```diff
@@ -0,0 +1,8 @@
+﻿using Training.TruckWorld.Backend.Domain.Entities;
+
+namespace Training.TruckWorld.Backend.Application.Emails.Interfaces;
```
Comment (Habiba67y) [inline src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Emails/Interfaces/IEmailSenderService.cs]: folderlar xatomasmi Notifications bo'limiga yozasiz manimcha

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Emails/Interfaces/IEmailSenderService.cs [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+public interface IEmailSenderService
+{
+    ValueTask<Email> SendEmailAsync(Email email);
```
Comment (Habiba67y) [inline src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Emails/Interfaces/IEmailSenderService.cs]: bo''ta ValueTask<bool> bo'miydimi?

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Infrastructure/Emails/Services/EmailSenderService.cs [hunk 3]
```diff
@@ -0,0 +1,48 @@
+﻿using System.Net;
+using System.Net.Mail;
+
+using Training.TruckWorld.Backend.Application.Emails.Interfaces;
+using Training.TruckWorld.Backend.Domain.Entities;
+
+namespace Training.TruckWorld.Backend.Infrastructure.Emails.Services;
```
Comment (Habiba67y) [inline src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Infrastructure/Emails/Services/EmailSenderService.cs]: bu yerda ham shunaqa

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Emails/Interfaces/IEmailSenderService.cs (hunk 2 above)
Comment (Habiba67y) [reply src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Emails/Interfaces/IEmailSenderService.cs]: ya'ni email ni o'rniga bool bo'miydimi ValueTaskni ichida

Comment (botirjonabdumannonovv) [reply src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Emails/Interfaces/IEmailSenderService.cs]: To'g'ri ,bool bo'ladi o'zgartirib qo'yaman

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Emails/Interfaces/IEmailSenderService.cs (hunk 1 above)
Comment (botirjonabdumannonovv) [reply src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Application/Emails/Interfaces/IEmailSenderService.cs]: ha hop hozir ko'chirib qo'yaman folderni

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Infrastructure/Emails/Services/EmailSenderService.cs (hunk 3 above)
Comment (botirjonabdumannonovv) [reply src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Infrastructure/Emails/Services/EmailSenderService.cs]: yani Notifications ga yozishim kerakmi?

**Comment by botirjonabdumannonovv** [comment]
- edited code and folder location of EmailSenderService

File: src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Infrastructure/Emails/Services/EmailSenderService.cs (hunk 3 above)
Comment (Habiba67y) [reply src/Training.TruckWorld.Clone.Backend/Training.TruckWorld.Backend/Infrastructure/Emails/Services/EmailSenderService.cs]: ha shunday

**Review by Habiba67y — APPROVED** [review]

**Review by Mirzohid-Mominov — APPROVED** [review]

**Pull request merged**